### PR TITLE
Refactored boilerplate to a function

### PR DESCRIPTION
**Pull request by hibooboo2**

### CODE REVIEW

File: tests/test_docker.py [hunk 1]
```diff
@@ -980,3 +818,20 @@ def container_field_test_boiler_plate(resp):
     del docker_container['Status']
     del fields['dockerIp']
     docker_container = _sort_ports(docker_container)
+
+
+def instance_activate_common_validation(resp):
```
Comment (cjellick) [inline tests/test_docker.py]: Take a look at the function right above this (```container_field_test_boiler_plate```)
Looks like your function should call that function and then do just a few more things on top of it

Comment (hibooboo2) [inline tests/test_docker.py]: FIxed.

File: tests/test_docker.py [hunk 2]
```diff
@@ -980,3 +818,15 @@ def container_field_test_boiler_plate(resp):
...
+        container_field_test_boiler_plate(resp)
```
Comment (cjellick) [inline tests/test_docker.py]: Looks like there are 8 whitespaces for the lines in this function. Should be 4.

Comment (hibooboo2) [inline tests/test_docker.py]: Well the file passes Flake8

Comment (hibooboo2) [inline tests/test_docker.py]: You are correct fixed @cjellick

**Comment by hibooboo2** [comment]
@cjellick I believe i fixed it. Had to rebase but all is good now.

**Comment by cjellick** [comment]
LGTM Ping @ibuildthecloud

**Pull request merged**